website/docs/plugin/framework/handling-data/terraform-concepts.mdx

```diff
@@ -0,0 +1,729 @@
+---
+page_title: 'Plugin Development - Framework: Handling Data - Terraform Concepts'
+description: >-
+  Configuration, Schemas, Attributes and Blocks.
+---
+
+# Configuration, Schemas, Attributes and Blocks
+
+Schemas specify the fields that a provider, resource or data source configuration can have.
+
+The fields defined within the schema are either attributes or blocks.
+
+Attributes and blocks within Terraform configuration have different syntax, either using, or omitting an equals sign in their definition, respectively.
+
+Providers, resources and data sources have their own type-specific schema, which use type-specific attributes and blocks.
+
+The following examples use a resource but the same schema attribute and block types are also defined with the schema for providers and data sources. The examples focus on type-related information, the `Optional` field is included just to provide working examples but refer to Schema (add link) for information about the other fields that can be defined within attributes and blocks.
+
+## Attributes
```

Comment:
Similarly in a followup for these, it might make sense to summarize that attributes set values and provide a quick table of descriptions -- leaving the nitty gritty details for the attributes page. 👍